include/deal.II/fe/fe_nedelec.h

```diff
@@ -324,6 +324,18 @@ class FE_Nedelec : public FE_PolyTensor<dim>
   virtual std::unique_ptr<FiniteElement<dim, dim>>
   clone() const override;
 
+  /**
+   * For a finite element of degree @p sub_degree < @p degree, we return a
```

Comment:
```suggestion
   * For a finite element of degree larger than @p sub_degree, we return a
```
I think this is easier to read.

Reply:
I agree that this is easier to understand. I pushed this change.

Would you like me to make the same modification to the description for the equivalent function in `FE_Q_Hierarchical`?

Reply:
> Would you like me to make the same modification to the description for the equivalent function in `FE_Q_Hierarchical`?

I think that this would be a nice improvement, so go for it! I suggest opening a separate PR for this change.

Reply:
Sounds good. Will do.